android-libs/src/androidTest/java/io/ooni/mk/androidTest/MKVersionTest.java

```diff
@@ -0,0 +1,22 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+package io.ooni.mk.androidTest;
+
+import com.google.common.truth.Truth;
+
+import org.junit.Test;
+
+import androidx.test.filters.SmallTest;
+
+import io.ooni.mk.MKVersion;
+
+@SmallTest public class MKVersionTest {
+    static {
+        System.loadLibrary("measurement_kit");
+    }
+
+    @Test public void perform() {
+        Truth.assertThat(MKVersion.getVersionMK()).isEqualTo("0.10.4");
```

Comment:
Are you going to update this assert in every release?

Reply:
That's a good question. The correct check would probably be to make sure that we're returned a non-null, non-empty string matching a regex. What do you think?

Reply:
that would be much better